src/surveys/components/ContactFormResponse.tsx

```diff
@@ -0,0 +1,131 @@
+import { h } from 'preact';
+import {
+  FormField,
+  FormSettings,
+  Question,
+  SurveyAnswer,
+  Theme,
+} from '../../types';
+import { Button, Header, Input } from '../../components';
+import { useState } from 'preact/hooks';
+import { LockIcon } from 'lucide-preact';
+import { hexToRgba } from '../../utils';
+import AnswerContainer from './AnswerContainer';
+
+type Props = {
+  question: Question;
+  theme?: Theme;
+  onAnswered: (answers: SurveyAnswer[]) => void;
+  submitText?: string;
+  label: string;
+  description: string;
+};
+
+export const ContactFormResponse = ({
+  label,
+  description,
+  question,
+  theme,
+  onAnswered,
+  submitText,
+}: Props) => {
+  const [inputValues, setInputValues] = useState<{ [key: string]: string }>({});
```

Comment:
@afolasope Nice! Can we use `key: ID` here instead? `option.id` could also be a number.